services/web/client/source/class/osparc/desktop/NavigationBar.js

```diff
@@ -259,6 +259,20 @@ qx.Class.define("osparc.desktop.NavigationBar", {
       activityWindow.add(new osparc.component.service.manager.ActivityManager());
       activityWindow.center();
       activityWindow.open();
+    },
+
+    __openIssueInfoDialog: function() {
+      const issueConfirmationWindow = new osparc.ui.window.Dialog("Information", null,
+        this.tr("To create an issue in GitHub, you must have an account in GitHub and be already logged-in.")
+      );
+      const contBtn = new qx.ui.toolbar.Button(this.tr("Continue"), "@FontAwesome5Solid/external-link-alt/12");
+      contBtn.addListener("execute", () => window.open(osparc.component.widget.NewGHIssue.getNewIssueUrl()), this);
+      const loginBtn = new qx.ui.toolbar.Button(this.tr("Log in in GitHub"), "@FontAwesome5Solid/external-link-alt/12");
+      loginBtn.addListener("execute", () => window.open("https://github.com/login"), this);
```

Comment:
Why not query github and only display window if fails ... 
Having to click continue every time we want to create an issue is not a very smooth UX

Reply:
Because we don't have access to the user's Github auth tokens from the execution context of our app/browser tab.
https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy
Is there a way to do this that I don't know?

Reply:
query -> if 500 display message . If not, then it is because user has already logged in .. right?

Reply:
I don't think it works like that. Querying from our side will always return the 500 page, as we cannot provide GitHub with the user's authentication tokens. If we open a new window and do the request, then the browser will set them if the user is logged in, but then we don't have access to that window's content from our side.

Reply:
What i mean is to query github site from front-end before open the new window

Reply:
The problem is that if you make a GET request to https://github.com/ITISFoundation/osparc-issues/issues/new, the request will be blocked by CORS policy